analysis/src/Cli.ml

```diff
@@ -2,31 +2,30 @@ let help =
   {|
 **Private CLI For rescript-vscode usage only**
 
-Examples:
+API examples:
   ./rescript-editor-analysis.exe complete src/MyFile.res 0 4 currentContent.res
   ./rescript-editor-analysis.exe definition src/MyFile.res 9 3
-  ./rescript-editor-analysis.exe dump src/MyFile.res src/MyFile2.res
   ./rescript-editor-analysis.exe documentSymbol src/Foo.res
   ./rescript-editor-analysis.exe hover src/MyFile.res 10 2
   ./rescript-editor-analysis.exe references src/MyFile.res 10 2
+
+Dev-time examples:
+  ./rescript-editor-analysis.exe dump src/MyFile.res src/MyFile2.res
   ./rescript-editor-analysis.exe test src/MyFile.res
 
-Note: coordinates are zero-based, so the first position is 0 0.
+Note: positions are zero-indexed (start at 0 0), following LSP.
+https://microsoft.github.io/language-server-protocol/specification#position
 
 Options:
-  complete: compute autocomplete for MyFile.res at line 0 and column 4,
+  completion: compute autocomplete for MyFile.res at line 0 and column 4,
```

Comment:
@cristianoc I've changed the name here so that now every command follows their LSP wording counterpart, e.g. `textDocument/definition` and `textDocument/completion`.